packages/workbox-webpack-plugin/src/lib/get-workbox-sw-imports.js

```diff
@@ -14,26 +14,39 @@
   limitations under the License.
 */
 
-const {getModuleUrl} = require('workbox-build');
+const {copyWorkboxLibraries, getModuleUrl} = require('workbox-build');
 
 /**
  * @param {Object} compilation The webpack compilation.
  * @param {Object} config The options passed to the plugin constructor.
- * config.excludeChunks may be modified by this function if
- * config.importWorkboxFrom is set to a chunk name.
+ * - config.excludeChunks may be modified by this function if
+ *   config.importWorkboxFrom is set to a chunk name.
```

Comment:
I don't know if it's just me, but I don't fully understand the 'may be modified by this function' part. What function and why does excludeChunks have to have a comment regarding importWorkboxFrom is a chunk, I would have assumed that would be expected.